test/Microsoft.Azure.Devices.Edge.Test.Common/DaemonConfiguration.cs

```diff
@@ -6,137 +6,84 @@ namespace Microsoft.Azure.Devices.Edge.Test.Common
     using System.Collections.Generic;
     using System.IO;
     using System.Linq;
-    using System.Text.RegularExpressions;
+    using System.Threading;
+    using System.Threading.Tasks;
     using Microsoft.Azure.Devices.Edge.Test.Common.Certs;
     using Microsoft.Azure.Devices.Edge.Util;
 
-    public struct ConfigFilePaths
-    {
-        public string Keyd;
-        public string Certd;
-        public string Identityd;
-        public string Edged;
-    }
-
     public class DaemonConfiguration
     {
-        public enum Service
-        {
-            Keyd,
-            Certd,
-            Identityd,
-            Edged
-        }
-
-        private struct Config
+        struct Config
         {
             public string ConfigPath;
-            public string PrincipalsPath;
-            public string Owner;
-            public uint Uid;
             public TomlDocument Document;
         }
 
         const string GlobalEndPoint = "https://global.azure-devices-provisioning.net";
-        private Dictionary<Service, Config> config;
+        Config config;
 
-        public DaemonConfiguration(ConfigFilePaths configFiles)
+        public DaemonConfiguration(string superTomlPath)
         {
-            this.config = new Dictionary<Service, Config>();
-
-            this.InitServiceConfig(Service.Keyd, configFiles.Keyd, "aziotks");
-            this.InitServiceConfig(Service.Certd, configFiles.Certd, "aziotcs");
-            this.InitServiceConfig(Service.Identityd, configFiles.Identityd, "aziotid");
-            this.InitServiceConfig(Service.Edged, configFiles.Edged, "iotedge");
+            Directory.CreateDirectory(Directory.GetParent(superTomlPath).FullName);
```

Comment:
We prob should clean this dir up after the test.

Reply:
Done